test/end-to-end/test_studio_supervisor.ps1

```diff
@@ -2,17 +2,7 @@ $cliVersion = ((hab --version) -split " ")[1]
 $env:HAB_STUDIO_SECRET_HAB_INTERNAL_BLDR_CHANNEL="dev"
 hab origin key generate $env:HAB_ORIGIN
 
-# Note we kill all containers before and after this test
-# At some point we can remove the BeforeAll but neet it now
-# for straggler test VMs
-
 Describe "Studio supervisor" {
-    BeforeAll {
```

Comment:
Would there still be a need to cleanup things prior just to make sure we don't have any zombies from prior runs? Wondering whether we might still need that.

Reply:
I made some changes so that we are more guaranteed to only stop the container that the current build spun up. While this means that concurrent e2e tests will not trample on each other it also opens the possibility that a cancelled build cancelled at the right time could leave an orphaned container. Given the nature of the e2e pipeline, canceled build are less likely to occur and I don't forsee this to be a problem but it is something we should be aware of.